### ISSUE TITLE
`https: true` in the options object of createLayer doesn't work with Torque

### ISSUE THREAD

**Issue by matallo** [CartoDB/cartodb.js#254]
had to use, instead:

```
tiler_protocol: "https",
tiler_port: 443,
```

here's the piece of code:

```
  cartodb.createLayer(map, {
    type: "torque",
    order: 1,
    options: {
      tiler_protocol: "https",
      tiler_port: 443,
      table_name: "export_281_29_copy",
      user_name: "secretsanta",
      cartocss: 'Map { -torque-frame-count:512; -torque-animation-duration:30; -torque-time-attribute:"confirm_time"; -torque-aggregation-function:"count(cartodb_id)"; -torque-resolution:2; -torque-data-aggregation:cumulative; }  #export_281_29_copy{   comp-op: multiply;   marker-fill-opacity: 0.9;   marker-line-color: #FFF;   marker-line-width: 1.5;   marker-line-opacity: 1;   marker-type: ellipse;   marker-width: 1;   marker-fill: #FB462F; } #export_281_29_copy[frame-offset=1] {  marker-width:2;  marker-fill-opacity:0.9;  } #export_281_29_copy[frame-offset=2] {  marker-width:3;  marker-fill-opacity:0.9;  } #export_281_29_copy[frame-offset=3] {  marker-width:4;  marker-fill-opacity:0.9;  }'
      }
  })
  .addTo(map);
```

**Comment by javisantana** [comment]
cc @fdansv this needs to be fixed in cartodb.js something around: https://github.com/CartoDB/cartodb.js/blob/develop/src/vis/layers.js#L106

**Comment by fdansv** [comment]
Assuming you're passing ```https``` as an option in the third parameter (not in the layer options), the problem lied in some default values required to apply the https property not being set in the data object. Passing ```data``` through [```normalizeOptions()```](https://github.com/CartoDB/cartodb.js/blob/develop/src/vis/layers.js#L60), like we do in the rest of layer registers, does the job and fixes this issue.

**Comment by fdansv** [comment]
This is now fixed as of #331
